pennylane_lightning/src/bindings/Bindings.hpp

```diff
@@ -166,7 +183,7 @@ auto allocateAlignedArray(size_t size, const pybind11::dtype &dt)
  */
 void registerArrayAlignmentBindings(py::module_ &m) {
     /* Add CPUMemoryModel enum class */
-    pybind11::enum_<CPUMemoryModel>(m, "CPUMemoryModel")
+    py::enum_<CPUMemoryModel>(m, "CPUMemoryModel")
```

Comment:
```suggestion
    py::enum_<CPUMemoryModel>(m, "CPUMemoryModel",  py::module_local())
```

To avoid the bindings registering the same ops globally and causing issues once imported.

Reply:
Maybe we can later separate common utils to their own binary modules later.

Reply:
Amintor included that in his new PR.